kfs-tax/src/main/resources/edu/arizona/kfs/module/tax/spring-tax.xml

```diff
@@ -1,36 +1,150 @@
+                <value>classpath:edu/arizona/kfs/module/tax/document/datadictionary/*.xml</value>
+            </list>
+        </property>
+        <property name="databaseRepositoryFilePaths">
+            <list>
+                <value>edu/arizona/kfs/module/tax/ojb-tax.xml</value>
+            </list>
+        </property>
+        <property name="jobNames">
+            <list>
+                <value>batchPayeeFormJob</value>
+                <value>payeeMasterExtractJob</value>
+            </list>
+        </property>
+        <property name="batchFileDirectories">
+          <list>
+            <value>${staging.directory}/1099</value>
+            <value>${reports.directory}/tax</value>
+            <value>${staging.directory}/tax</value>
+          </list>
+        </property>
+    </bean>
+
+    <bean id="taxModuleService" parent="taxModuleService-parentBean" />
+    <bean id="taxModuleService-parentBean" class="org.kuali.kfs.sys.service.impl.KfsModuleServiceImpl" abstract="true">
+        <property name="moduleConfiguration" ref="taxModuleConfiguration" />
+    </bean>
+
+    <bean id="payeeMasterExtractJob" parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="payeeMasterExtractStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="batchPayeeFormJob" parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="batchPayeeFormStep" />
+                <ref bean="electronicFilingStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="payeeMasterExtractStep" class="edu.arizona.kfs.module.tax.batch.PayeeMasterExtractStep" parent="step">
+        <property name="taxBatchExtractService" ref="taxBatchExtractService" />
+        <property name="taxParameterHelperService" ref="taxParameterHelperService" />
+        <property name="businessObjectService" ref="businessObjectService" />
+    </bean>
+
+    <bean id="batchPayeeFormStep" class="edu.arizona.kfs.module.tax.batch.BatchPayeeFormStep" parent="step">
+        <property name="taxPayeeService" ref="taxPayeeService"></property>
+        <property name="taxParameterHelperService" ref="taxParameterHelperService"></property>
+        <property name="taxForm1099GeneratorService" ref="taxForm1099GeneratorService"></property>
+    </bean>
+
+    <bean id="electronicFilingStep" class="edu.arizona.kfs.module.tax.batch.ElectronicFilingStep" parent="step">
+        <property name="electronicFilingService" ref="electronicFilingService" />
+        <property name="taxParameterHelperService" ref="taxParameterHelperService" />
+        <property name="taxForm1099GeneratorService" ref="taxForm1099GeneratorService" />
+    </bean>
+
+    <bean id="taxReportingDao" parent="platformAwareDao" class="edu.arizona.kfs.module.tax.dataaccess.impl.TaxReportingDaoImpl">

```

Comment:
Why is this class named TaxReportingDaoImpl.java? DAO impl class names in KFS usually end with either Ojb or Jdbc. This class name should be TaxReportingDaoOjb.java